### ISSUE TITLE
GPS button doesn’t work?

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#1103]

**Comment by fancyham** [comment]
- duplicate of #1101

**Comment by Benbaillou** [comment]
Issue to be closed as the duplicate is already in columns "Bugs"

**Comment by ryu-jieun** [comment]
I think this issue is directly related to #1101?
